reloj-checador/app/javascript/packs/layouts/navbar.js

```diff
@@ -0,0 +1,14 @@
+controller_page = (document.getElementById("controller_name").value).toString();
+switch (controller_page) {
+    case 'attendaces':
+        document.getElementById("attendaces_li").classList.add('active');
+        break;
+    case 'companies':
+        document.getElementById("companies_li").classList.add('active');
+        break;
+    case 'employees':
+        document.getElementById("employess_li").classList.add('active');
+        break;
+    default:
+        break;
+}
```

Comment:
WDYT, about simplifying this decision, with an if, like the following:

```
if (controller_page == 'attendaces' || controller_page == 'companies' || controller_page == 'employees' ) {
	document.getElementById(controller_page + "_li").classList.add('active');
}
```

Reply:
Taking into consideration the sugestion, the code now is:
`if (['attendaces','companies','employees','reports'].indexOf(controller_page) >= 0 ) {
	document.getElementById(controller_page + "_li").classList.add('active');
}`